packages/ui/src/2_molecules/NavMenuItem/NavMenuItem.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, ReactNode, useMemo, useRef } from 'react';
+
+import classNames from 'classnames';
+
+import { Badge } from '../../1_atoms/Badge/Badge';
+import styles from './NavMenuItem.module.css';
+
+const MAX_NOTIFICATIONS_COUNT = 99;
+
+type NavMenuItemProps = {
+  count?: number;
+  onClick?: () => void;
+  children: ReactNode;
+  isActive?: boolean;
+  className?: string;
+  dataActionId?: string;
+};
+
+export const NavMenuItem: FC<NavMenuItemProps> = ({
+  count,
+  onClick,
+  children,
+  isActive,
+  className,
+  dataActionId,
+}) => {
+  const ref = useRef<HTMLButtonElement>(null);
```

Comment:
```suggestion

```
This `ref` is useless as you do nothing with it, you just assign it to the button. What you should do is to use `forwardRef` and expose `ref` this way but it will be done in bulk in the upcoming sprint so you don't need to add it now.